docs/requests.md

```diff
@@ -21,6 +16,30 @@ You need to have a working version of Chromedriver http://chromedriver.chromium.
       A-->H[Email Print];
 ```
 
+* [Borrow Direct](https://catalog.princeton.edu/borrow-direct)
+  Can cancel requests by connecting to the URL above
+  * Used to request items from partners.  Only connects for unavailable items that include an ISBN number
+* Illiad
+  Can cancel request from [orangelight](https://catalog.princeton.edu/account/digitization_requests)
+  * Used to request unavailable items that are not borrow direct eligible.
+  * Used to request Digitizations
+* [Alma](https://princeton.alma.exlibrisgroup.com/discovery/account?vid=01PRI_INST:Services&lang=EN&section=overview)
+  can cancel request by connecting to the URL above
+  * Used to request pick-up of available items on the shelf
+  * Holds are created for ReCAP Items when physical delivery is requested
+  * Holds are requested for Marquand Offsite (clancy) items when physical or digital item is requested
+* Clancy (ciasoft)
+  All requests on qa and staging go to a test system, so they do not need to be canceled
+  * Used to request items from Marquand
+    1. Check if items are present in Clancy (all Marquand items)
```

Comment:
Do we do this check even if the Marquand items are in a ReACP location? Those would not be managed through the Clancy facility.

Reply:
Ah no good point @kevinreiss I was assuming those items were considered part of ReCAP.  I'll add a note...